docs/template_plugin/tests/functional/op_reference/lstm_cell.cpp

```diff
@@ -34,6 +28,22 @@ struct LSTMCellParams {
     std::string testcaseName;
 };
 
+struct Builder : ParamsBuilder<LSTMCellParams> {
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, batchSize);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, inputSize);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, hiddenSize);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, gatesCount);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, X);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, W);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, R);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, H_t);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, C_t);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, B);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, Ho);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, Co);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, testcaseName);
+};
+
 class ReferenceLSTMCellTest : public testing::TestWithParam<LSTMCellParams>, public CommonReferenceTest {
```

Comment:
Could you please add LSTM-5 tests?

Reply:
Did you mean LSTMCell-4?
This PR added LSTMCell-1 test cases to those of the LSTMCell-4 in https://github.com/openvinotoolkit/openvino/pull/8314.